### DISCUSSION TITLE
Need help to scrap (XPATH) this web page www.colibris-lemouvement.org

### DISCUSSION THREAD

**Discussion by Fluidetom** [FreshRSS/FreshRSS#6501] [Q&A] [answered]
Hey Guys,

I don't know if this is the right place for this kind of request, but I'm having some issue to scrape a particular website.
I already have many XPATH type feeds in my FreshRSS instance, but this one is causing me a headache.
If you don't mind, could you guys take a look at my code so see what could be blocking a successful XPATH scraping?
FreshRSS keeps sending me back "This feed encountered an issue", but I cannot figure out what is wrong.

Thanks in advance

**Feed URL**
`https://www.colibris-lemouvement.org/magazine/recherche?rub=67`

**Feed title**
`"Colibirs"`

**XPath for finding news items**
`//li[@class="item double" or @class="item single"]`

**XPath for title**
`descendant::div[@class="text"]/h2`

**XPath for item content**
`descendant::div[@class="field-teaser"]`

**XPath for item link (URL)**
`descendant::div[@class="text"]/h2/a/@href`

**XPath for thumbnail**
`descendant::div[@class="field-image"]/img/@src`

**XPath for author** (I'd like to use the news category instead)
`descendant::span[@class="rubs"]/a`

**Comment by Frenzie** [comment]
They have some feed btw
https://www.colibris-lemouvement.org/taxonomy/term/105/feed (at the bottom on https://www.colibris-lemouvement.org/projet/publications)

**Reply by Fluidetom** [reply]
Thanks @Frenzie , good find.
I had a look, but this feed does work, but does not include all the publications from their website.
The page I was trying to scrape contains many more articles :  https://www.colibris-lemouvement.org/magazine/recherche?rub=67.

**Reply by Frenzie** [reply, marked as answer]
The /taxonomy/term can be followed by other numbers (like 67), https://www.colibris-lemouvement.org/taxonomy/term/67/feed

**Reply by Fluidetom** [reply]
**You're a genius**, this is exactly what I was looking for !
Thanks a bunch! Much appreciated.

May I ask how did you find that out ?
Is there a (not too complicated) way to find that kind of info ?
I'm not a web dev, but have good tech skills nonetheless.

**Reply by Frenzie** [reply]
In retrospect that path is probably just [standard Drupal](https://www.drupal.org/forum/support/post-installation/2012-09-03/rss-feed-url-locations), but my process wasn't quite that informed. I took a quick look around, found the feed, and, well, a number is a number.

Wordpress websites are simpler: you can directly add `/feed` to the end of most pages.
